### PR TITLE
Forget password feature is delete and tested 

### PR DESCRIPTION
**Pull request by chiragkhimani**
Everything good and tested based on client requirement

### CODE REVIEW

**Comment by chiragkhimani** [comment]
Fantastic work

**Comment by MayramShadieva** [comment]
Thank you

**Comment by EnesErduran** [comment]
Looks nice

File: src/com/java/gitpractice/HomePage.java [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 public class HomePage {
 	public static void main(String[] args) {
-		System.out.println("logo");
+		System.out.println("Welcome to Ebay");
```
Comment (chiragkhimani) [inline src/com/java/gitpractice/HomePage.java:5]: Please make ebay capital like EBAY

Comment (sagambay) [reply src/com/java/gitpractice/HomePage.java:5]: where is this part?

Comment (chiragkhimani) [reply src/com/java/gitpractice/HomePage.java:5]: It's line 5

**Pull request merged**